lib/utils.js

```diff
@@ -1927,12 +1929,14 @@ module.exports.nodeMajorVersion = function() {
  * @param {Object} parameters the parameters to format into the template
  * @returns {string} the formatted file path
```

Comment:
Need to document the project and filetype parameters. These should be considered optional and only needed if you want to use the [resourceDir] replacement

Reply:
I added a description regarding the parameters. Please check if it's appropriate.